scripts/commit_classifier.py

```diff
@@ -552,31 +577,12 @@ def feature_sort_key(f):
     def classify(
         self,
         revision=None,
-        phabricator_deployment=None,
-        diff_id=None,
         runnable_jobs_path=None,
     ):
         if revision is not None:
             assert phabricator_deployment is None
             assert diff_id is None
 
-        if diff_id is not None:
-            assert phabricator_deployment is not None
-            assert revision is None
-
-        self.update_commit_db()
-
-        with hglib.open(self.repo_dir) as hg:
-            if phabricator_deployment is not None and diff_id is not None:
-                self.apply_phab(hg, phabricator_deployment, diff_id)
-
-                revision = hg.log(revrange="not public()")[0].node.decode("utf-8")
-
-            # Analyze patch.
-            commits = repository.download_commits(
-                self.repo_dir, rev_start=revision, save=False
-            )
```

Comment:
This won't work, as the `commits` variable is used later in the function.

Reply:
I got my mistake, resolved it.